src/news/views.py

```diff
@@ -23,11 +23,17 @@ def index(request, language_code):  # pylint: disable=W0613
         end = (count * page)
         posts = posts[start:end]
     for item in posts:
-        result.append({'id': item.id, 'title': item.title, 'tags': [item.name for item in item.newscategory.all()], 'date': str(item.pub_date)})
+        result.append({'id': item.id, 'title': item.title,
+                       'tags': [item.name for item in item.newscategory.all()],
+                       'date': str(item.pub_date)})
     result_json = json.dumps(result)
     return HttpResponse(result_json, content_type="application/json")
```

Comment:
why not use JsonResponse directly?

Reply:
No reason at all. Just happened this way and changing it totally valid :)